exonum-java-binding/core/rust/src/runtime/cmd.rs

```diff
@@ -114,6 +128,39 @@ pub struct Run {
     jvm_args_append: Vec<String>,
 }
 
+/// EJB-specific `run-dev` command.
+///
+/// Automatically generates node configuration for one
+/// validator and runs it using provided `artifacts_dir` as a directory for configuration
+/// files, database and Java service artifacts.
+#[derive(Debug, StructOpt, Serialize, Deserialize)]
+#[structopt(rename_all = "kebab-case")]
+pub struct RunDev {
+    /// Path to the directory containing Java service artifacts and temporary files for
+    /// the node.
+    #[structopt(long)]
+    artifacts_dir: PathBuf,
+}
+
+impl RunDev {
+    /// Returns to the file or directory in the artifacts directory.
+    fn artifact_path(&self, artifact_name: &str) -> PathBuf {
+        let mut path = self.artifacts_dir.clone();
+        path.push(artifact_name);
+        path
+    }
+
+    /// Deletes database directory if it exists.
+    fn cleanup(&self) -> Result<(), failure::Error> {
```

Comment:
I don't want this command to remove some random directories that appear in the artifacts path.